backends/verilog/verilog_backend.cc

```diff
@@ -1885,6 +1943,11 @@ struct VerilogBackend : public Backend {
 				nostr = true;
 				continue;
 			}
+			if (arg == "-extmem") {
+				extmem = true;
```

Comment:
I think there's an `extmem = false;` default assignment missing at the start of this function.

Reply:
Thanks, fixed